.github/workflows/auto-approve.yml

```diff
@@ -14,14 +14,14 @@ jobs:
   tidy:
     name: run go mod tidy and updated
     runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
     steps:
       - name: checkout
         uses: actions/checkout@v1
         if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
       - name: reattach HEAD to Head Ref
         # b/c checkout action leaves in detached head state https://github.com/actions/checkout/issues/6
         run: git checkout "$(echo ${{ github.head_ref }})"
-        if: github.head_ref != '' && (github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]')
```

Comment:
You may still want to retain `if: github.head_ref != ''` on this line.

Reply:
darn it

Reply:
ok. getting rid of it of the previous if and adding this one back in.